kobo/apps/openrosa/apps/logger/tests/bulk_submission/test_bulk_submission_attachments.py

```diff
@@ -0,0 +1,87 @@
+# coding: utf-8
+import os
+
+from django.urls import reverse
+
+from kobo.apps.openrosa.apps.logger.import_tools import import_instances_from_zip
+from kobo.apps.openrosa.apps.logger.models import Instance
+from kobo.apps.openrosa.apps.logger.views import bulksubmission
+from kobo.apps.openrosa.apps.main.tests.test_base import TestBase
+from kobo.apps.openrosa.libs.utils.storage import rmdir
+from kpi.deployment_backends.kc_access.storage import (
+    default_kobocat_storage as default_storage,
+)
+
+CUR_PATH = os.path.abspath(__file__)
+CUR_DIR = os.path.dirname(CUR_PATH)
+
+ASSET_UID = 'aLf54MbS5yrNQjwPkLKDms'
+XLSX_FILE_PATH = os.path.join(CUR_DIR, ASSET_UID + '.xlsx')
+ZIP_FILE_PATH = os.path.join(CUR_DIR, ASSET_UID + '.zip')
+
+
+class TestBulkSubmissionAttachments(TestBase):
+
+    def setUp(self):
+        TestBase.setUp(self)
+        self._publish_xls_file(XLSX_FILE_PATH)
+
+    def _stored_attachment_basenames(self, instance):
+        placeholder_path = '{username}/attachments/{xform_uuid}/{instance_uuid}'
+        attachments_path = placeholder_path.format(
+            username=self.user.username,
+            xform_uuid=instance.xform.uuid,
+            instance_uuid=instance.uuid,
+        )
+        _, attachments = default_storage.listdir(attachments_path)
+        return attachments
+
+    def tearDown(self):
+        # delete everything we imported
+        Instance.objects.all().delete()
+        if self.user and self.user.username:
+            rmdir(self.user.username)
+
+    def test_bulk_import_attachments_zip(self):
+        self.assertFalse(Instance.objects.exists())
+
+        import_instances_from_zip(ZIP_FILE_PATH, self.user)
+
+        queryset = Instance.objects
+
+        self.assertEqual(queryset.count(), 2)
+
+        basenames_count = 0
+        for instance in queryset.all():
+
+            # Check the database
+            expected = [
+                ('test.pdf', 'application/pdf'),
+                ('thanks.png', 'image/png'),
+                ('wave.wav', 'audio/x-wav'),
+            ]
```

Comment:
Related to https://github.com/kobotoolbox/kpi/pull/6055/files#r2263019446, maybe we should include an audit CSV and background audio recording. Will leave the decision to @noliveleger as to whether it's worth delaying this fix for that.

I'll "request changes" here, but it's really because I want the `xform_fs` discussion to be resolved before merge.

Reply:
Since including files is no longer contingent on XML parsing, I'm fine not having additional tests at this time